glass-easel-template-compiler/src/js_bindings.rs

```diff
@@ -12,6 +12,26 @@ pub struct TmplGroup {
     names: Vec<String>,
 }
 
+impl From<TmplError> for JsValue {
```

Comment:
Not reasonable. Why?

Reply:
It will be more reasonable if I can implement it only for `Error` in JS. I'll revert it back to the original implementation